back-end/department/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from basemodel.models import BaseModel
+from django.contrib.auth.models import User
+
+
+# Create your models here
+
+class Department(BaseModel):
+    name = models.TextField(blank=True,null=True,max_length=50)
+    shortcut = models.TextField(blank=True,null=True,max_length=10)
```

Comment:
Not necessarily required. If essentially required, use an enum.

Reply:
Shall I resolve converation

Reply:
Should I remove shortcut field. Not possible to use enum as number of choices changes as new departments are added

Reply:
Yes, remove it

Reply:
I think we need Shortcut such as CSE EE etc and full department name Computer Science, Electrical so that we can use shortcut in applications and full name in website when we show  it in front page

Reply:
That can be held on the frontend. But it doesn't take up much space anyway, can keep it.